### DISCUSSION TITLE
Question about bearware.dk web logins

### DISCUSSION THREAD

**Discussion by Mchopra7** [BearWare/TeamTalk5#1019] [Q&A]
Hello. I was wondering about bearware.dk logins on servers. Can they be set as administrators on a TeamTalk server?

**Comment by bear101** [comment]
Yes, create a user account with your BearWare.dk weblogin ID followed by @bearware.dk and assign this user as administrator.

E.g. on the public TeamTalk server I have a user account with the username bear_dk@bearware.dk. This user account is administrator.

**Comment by bear101** [comment]
More information on user accounts can be found [here](https://bearware.dk/teamtalk/v5.8.1/docs/userinterface.html#useraccountsdlg).
